### ISSUE TITLE
[JSSCP] Monthly Animal Health Worker Reporting form

### ISSUE THREAD

**Issue by utkarshavni** [avniproject/JSSCP#258]
**Motivation:**
Cluster Coordinator in each cluster is required to compile details on a monthly basis for Animal health from Animal Health Workers which is useful in monthly reporting of Animal health activities in the cluster.

**Goal:** 
When Monthly Monitoring Subject is Registered by user, Monthly Animal Health Worker Reporting encounter should be scheduled on the next 25th. Thereafter the form is scheduled on every 25th of the month for monthly reporting. The user is be able to record all the details as required for the monthly form as mentioned in the requirements below.

**Form link:**
https://docs.google.com/spreadsheets/d/1W3TPYyp9F5GS-BHaYvy8eCwUqVdQp9bcHKMBtT-w2G0/edit#gid=836611880

**Acceptance Criteria:**
The encounter must get scheduled on 25th of every month after Monthly monitoring Subject is registered. 
Cluster coordinator should be able to complete the form successfully by filling in all the details.
Wherever Total is mentioned, it should be auto calculated and Read only field.

**Comment by AchalaBelokar** [comment]
Visit scheduling is not working

**Comment by AchalaBelokar** [comment]
I am feeling the data on the date of 28 Aug 2022, after this visit should be scheduled on 25 Sept 2022. but currently, it is scheduling on 25th of august

**Comment by Gojo-Taqi** [comment]
- I have registered a monthly monitoring called peter and made an unplanned visit at 2-Aug which was scheduled at 25-Aug. The next visit was scheduled at 25-September like it should have.
- Then I did a visit at 27-Aug which was scheduled at 25-Aug, and the next visit was scheduled at 25-September like it should.
Not sure why it's not working for you.

![Screenshot from 2022-08-02 12-22-18.png](https://images.zenhubusercontent.com/627c9e38b2b7107ec1888b53/a18dae9b-9163-4188-b60c-d83d50a02395)
![Screenshot from 2022-08-02 12-29-16.png](https://images.zenhubusercontent.com/627c9e38b2b7107ec1888b53/f8032e87-4d5e-4b6a-8a3e-0a6db9fddb42)
